From 68b0591cf37760e09e358533bbcecf36eddfceed Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Sun, 10 Feb 2019 14:46:41 +0100
Subject: [PATCH] src/Makefile.am: add missing @RT_LIB@ for sg_turs

The sg_turs program uses clock_gettime(), so it should link against
librt, as provided by @RT_LIB@.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 881cf29..0eba680 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -170,7 +170,7 @@ sg_test_rwbuf_LDADD = ../lib/libsgutils2.la
 
 sg_timestamp_LDADD = ../lib/libsgutils2.la
 
-sg_turs_LDADD = ../lib/libsgutils2.la
+sg_turs_LDADD = ../lib/libsgutils2.la @RT_LIB@
 
 sg_unmap_LDADD = ../lib/libsgutils2.la
 
-- 
2.20.1

